Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -104,6 +105,24 @@ public enum TypeDescription: Codable, Hashable {
       } else {
         return "\(parentType.asSource).\(name)<\(generics.map { $0.asSource }.joined(separator: ", "))>"
       }
+    case let .attributed(type, specifier: specifier, attributes: attributes):
+      func attributesFromList(_ attributes: [String]) -> String {
+        attributes
+          .map { "@\($0)" }
+          .joined(separator: " ")
+      }
+      switch (specifier, attributes) {
+      case let (.some(specifier), .none):
+        return "\(specifier) \(type.asSource)"
+      case let (.none, .some(attributes)):
+        return "\(attributesFromList(attributes)) \(type.asSource)"
+      case let (.some(specifier), .some(attributes)):
+        // This case likely represents an error.
```

Comment:
I'm not _convinced_ of this but I've been having trouble coming up with legitimate Swift syntax where we have both a specifier and attributes

Reply:
Note that because I believe this is an error I haven't been able to write a test case that covers this case. Same goes for the case below.

Reply:
Note that because I believe this is an error I haven't been able to write a test case that covers this case.

Reply:
Let's add more context to this comment, i.e. that we can't figure out compiling code.

Reply:
addressed in af09cb1